nipype/interfaces/ants/registration.py

```diff
@@ -17,6 +17,11 @@
 from ..base import TraitedSpec, File, Str, traits, InputMultiPath, isdefined
 from .base import ANTSCommand, ANTSCommandInputSpec
 
+try:
+    LOCAL_DEFAULT_NUMBER_OF_THREADS = int(os.getenv("LOCAL_DEFAULT_NUMBER_OF_THREADS"))
+except TypeError:
+    LOCAL_DEFAULT_NUMBER_OF_THREADS = 1
```

Comment:
Let's just import `from .base import LOCAL_DEFAULT_NUMBER_OF_THREADS`.

Also, I think in general this variable is kind of pointless, unless somebody's using a fork of nipype where it's defined as something other than 1. This isn't something that needs to be fixed in this PR, but just to flag it for others' attention.